pkg/executor/core/executor.go

```diff
@@ -51,8 +53,17 @@ type Executor struct {
 }
 
 // TODO Logging within executor
+// VerifySubject verifies the subject and returns results.
 func (executor Executor) VerifySubject(ctx context.Context, verifyParameters e.VerifyParameters) (types.VerifyResult, error) {
-	result, err := executor.verifySubjectInternal(ctx, verifyParameters)
+	if featureflag.UseRegoPolicy.Enabled {
+		return executor.verifySubjectForRegoPolicy(ctx, verifyParameters)
```

Comment:
based on the verifier report, does customer have ability to see if report was generated through the rego path vs json policy? Maybe we could add some logging for debugging scenarios.